tests/unit/core/components/TestInstallButton.js

```diff
@@ -205,6 +205,38 @@ describe(__filename, () => {
     expect(button).toHaveProp('disabled', true);
   });
 
+  it('renders a switch button if useButton is false', () => {
+    const root = render({
+      addon: createInternalAddon({
```

Comment:
Why is this needed?

Reply:
Turns out: it's not! Copied from another test and assumed it was needed, but it wasn't.